modules/gke/variables.tf

```diff
@@ -8,6 +8,12 @@ variable "region" {
   description = "The region to create the GKE cluster in"
 }
 
+variable "enabled" {
+  type = bool
+  default = true
```

Comment:
no point of having a default value in submodules imo, only for top-level variables defined by the users.

Reply:
Very good point, good catch thanks 👍🏻